src/main/app/src/app/signaleringen/signaleringen-settings/signaleringen-settings.component.html

```diff
@@ -0,0 +1,37 @@
+<!--
+  ~ SPDX-FileCopyrightText: 2022 Atos
+  ~ SPDX-License-Identifier: EUPL-1.2+
+  -->
+
+<div class="table-container">
+    <div class="table-loading-shade" *ngIf="isLoadingResults"></div>
+    <div class="table-wrapper">
+        <table mat-table [dataSource]="dataSource"
+               matSort #instellingenTable
+               matSortDisableClear="true"
+               matSortActive="type"
+               matSortDirection="asc">
+            <ng-container [matColumnDef]="column" *ngFor="let column of columns">
+                <th mat-header-cell *matHeaderCellDef mat-sort-header>
+                    {{'signalering.' + column | translate}}
+                </th>
+                <td mat-cell *matCellDef="let row">
+                    <ng-container *ngIf="column == 'type' || column == 'subjecttype'">
+                        {{'signalering.' + column + '.' + row[column] | translate | empty}}
+                    </ng-container>
+                    <ng-container *ngIf="column != 'type' && column != 'subjecttype'">
+                        <mat-checkbox (click)="$event.stopPropagation()"
+                                      (change)="$event ? changed(row,column,$event.checked) : null"
```

Comment:
Check op $event omdat hij vanuit typescript ook gewijzigd wordt?

Reply:
Ah, is dat waar het voor is? OK dan haal ik hem weg.